commands/command_unlock.go

```diff
@@ -65,9 +65,15 @@ func unlockCommand(cmd *cobra.Command, args []string) {
 
 	locks := make([]unlockResponse, 0, len(args))
 	success := true
+	lockData, err := computeLockData()
+	if err != nil {
+		lockClient.Close()
+		Error(err.Error())
+		os.Exit(3)
+	}
```

Comment:
All the thoughts I had about this construction in the `commands/command_lock.go` file apply here too, of course.